newIDE/app/public/index.html

```diff
@@ -61,5 +61,29 @@
         </p>
       </div>
     </noscript>
+    <script>
+      (function() {
+        // Apply background color for startup
+        var backgroundColors = {
+          Dark: '#252525',
+          Nord: '#2E3440',
+          'Solarized Dark': '#00212B'
+        };
+        var themeName;
+        try {
+          themeName = JSON.parse(localStorage.getItem("gd-preferences"))['themeName'];
+        } catch (err) {
+          console.error(err);
```

Comment:
Include a detail error message in addition to the raw error

Reply:
This is not addressed

Reply:
I think this was the only one I addressed in this commit. Can you be more specific by what you meant be a detailed error message?

Reply:
Logging just the exception is not sufficient to understand the issue for someone looking at the console. Do something like:

`console.error("Unable to parse the preferences to set up the background color:", err)`

so that it's clear what went wrong.

Reply:
Ok, that makes more sense.

Reply:
@4ian Done.